openstack/manila/templates/etc-configmap.yaml

```diff
@@ -14,7 +14,7 @@ data:
 {{ include (print .Template.BasePath "/etc/_manila.conf.tpl") . | indent 4 }}
   api-paste.ini: |
 {{ include (print .Template.BasePath "/etc/_api-paste.ini.tpl") . | indent 4 }}
-  policy.json: |
+  policy.yaml: |
 {{ include (print .Template.BasePath "/etc/_manila-policy.json.tpl") . | indent 4 }}
```

Comment:
manila-policy template should be renamed and converted to yaml
+ there seem to be a couple of changes in the policies according to: https://docs.openstack.org/releasenotes/manila/de/wallaby.html